test/smoke/testApps/CoreAndFilter/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/CoreAndFilterTests.java

```diff
@@ -259,4 +259,21 @@ public void testRequestSlowWithResponseTime() {
         long expected = (new Duration(0, 0, 0, 20, 0).getTotalMilliseconds());
         assertTrue(actual >= expected);
     }
+
+    @Test
+    @TargetUri("/autoExceptionWithFailedRequest")
+    public void testAutoExceptionWithFailedRequest() {
```

Comment:
@Terryxxx, please mark this test with `@Ignore` and uncomment lines 270--271. I want to be sure we see that this test is being ignored.

Also, make a comment referencing bug #600. When we fix that bug, this test should pass.

Reply:
@littleaj Thanks for your comment. I will finish it today.